src/gui/SettingsWidgetGeneral.ui

```diff
@@ -106,6 +106,13 @@
         </widget>
        </item>
        <item>
+        <widget class="QCheckBox" name="ignoreGroupExpansionCheckBox">
+         <property name="text">
+          <string>Do not mark a database as modified when groups are expanded</string>
```

Comment:
@droidmonkey I updated the text for the new option. I don't think there's any other change that would qualify as non-data change at the moment. Maybe reordering groups, but then again it's not something that a user does when simply looking for a password, so I think it should still mark the db as dirty.